pkg/rekor/identity.go

```diff
@@ -94,8 +107,22 @@ func (e *IdentityEntry) String() string {
 	return strings.Join(parts, " ")
 }
 
+// ConcatOIDMatchers groups all OID matchers from OIDMatchers, FulcioExtensions, and CustomOIDs into one slice
+func ConcatOIDMatchers(mvs MonitoredValues) ([]OIDMatcher, error) {
+	FulcioOIDMatchers, err := RenderOIDMatchers(mvs.FulcioExtensions)
+	if err != nil {
+		return nil, fmt.Errorf("error rendering OID Matchers from Fulcio OID extensions: %w", err)
+	}
+	return slices.Concat(mvs.OIDMatchers, FulcioOIDMatchers), nil
```

Comment:
Should we merge based on OID? This code would mean we would have duplicate OIDs in the array, which would still function fine based on how this is used, but I think having a mapping of one OID to many values would be easier to test.